charts/cloudflow-provisioning/values.yaml

```diff
@@ -0,0 +1,22 @@
+tenantId: ''
+cloudflowName: ''
+region: us-east-2
+ami: ami-073c12bdffb704056
+instanceType:
+  support: m5a.xlarge
+  gpu: g4dn.xlarge
```

Comment:
g4dn.2xlarge

Reply:
👍 I just put this as a default value, these will be overrode by crossplane during use.